dor-workflow-service.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
 
   gem.add_dependency 'activesupport', '>= 3.2.1', '< 5'
-  gem.add_dependency 'nokogiri', '~> 1.6.0'
+  gem.add_dependency 'nokogiri', '~> 1.7'
```

Comment:
Won't going to 1.7 force a ruby upgrade to 2.1? https://github.com/sparklemotion/nokogiri/blob/master/CHANGELOG.md#backwards-incompatibilities-1